### ISSUE TITLE
Import Firefox 68 schema

### ISSUE THREAD

**Issue by EnTeQuAk** [mozilla/addons-linter#2623]
There is already a `FIREFOX_68_0b6_RELEASE` tag so we should import the schema as soon as possible.

**Comment by EnTeQuAk** [comment]
QA: There's various changes in the Webextension API as you can see in the pull request.

A few things I noted that may be relative easy to test to verify this works as expected are:

* There's now a `dark_theme` type for static themes
* The whole new `captivePortal` API seems to be completely undocumented for now, at least there is now a new `captivePortal` permission that can be used and a new `captivePortal` API that can be used, e.g something like…

```javascript
document.captivePortal.onStateChanged.addListener(details => {
	console.log('XXXXXXXXXXX');
});
```

should work I think.

Before this schema change:

```
    "warnings": [
        {
            "_type": "warning",
            "code": "UNSUPPORTED_API",
            "message": "captivePortal.onStateChanged is not supported",
            "description": "This API has not been implemented by Firefox.",
            "column": 1,
            "file": "background.js",
            "line": 7
        }
    ]
```

After this schema change, that code doesn't issue a warning anymore.

**Comment by AlexandraMoga** [comment]
Verified on -dev with Fx67. Win10x64

First, I've uploaded several add-ons of various types to see if there are any regressions. This part is working as before

@EnTeQuAk regarding the new supported properties:

- `dark_theme` doesn't seem to be recognized as a theme type. My understanding was that `dark_theme` should be identified the same as `theme` inside the manifest. 
However, if i'm trying to upload a theme with this new property, it ends up being identified as an extension.
Examples: 
https://addons-dev.allizom.org/en-US/firefox/addon/trans-blue/
https://addons-dev.allizom.org/en-US/firefox/addon/fire-and-water-68/

- `captivePortal` as a permission inside the manifest:
when used in prod, this permission will throw a warning: `/permissions: Unknown permissions "captivePortal"`. In -dev, this warning is no longer triggered, meaning that it is being recognized. 

- as for the API error generated when `captivePortal.onStateChanged` is used, I wasn't able to trigger it on prod, therefore I can't validate if this function is accepted on -dev after the schema import

**Comment by EnTeQuAk** [comment]
Thanks for testing this!

Regarding the `dark_theme` part, if you were able to submit it, that already verifies that it's working. That it's not being identified as a theme is a bug on addons-server, mind filing one? We can fix it there, given that it's a very recent feature I think it's fine to fix that later.

>  as for the API error generated when captivePortal.onStateChanged is used, I wasn't able to trigger it on prod, therefore I can't validate if this function is accepted on -dev after the schema import

This one is hard, I verified it using a local addons-linter build so I think that should be fine for now. As there isn't any documentation out there it's hard to see how it's supposed to be used.

> captivePortal as a permission inside the manifest:
when used in prod, this permission will throw a warning: /permissions: Unknown permissions "captivePortal". In -dev, this warning is no longer triggered, meaning that it is being recognized.

Perfect!

**Comment by AlexandraMoga** [comment]
Thanks!
I've filed https://github.com/mozilla/addons-server/issues/11593 as a follow up for the `dark_theme` issue
